front-end/src/app/interview/list-interview/list-interview.component.html

```diff
@@ -1,68 +1,69 @@
       <table class="table table-striped custab" *ngIf="list_interview?.length > 0 ; else elseBlock">
-        <tr>
-          <th scope="row">Interviews</th>
-          <th scope="row">Date of the Interview</th>
-          <th scope="row">Comments</th>
-          <th scope="row">Actions</th>
-        </tr>
-        <tr *ngFor="let item of list_interview">
-          <td>{{item.statusCandidate.name}}</td>
-          <td>{{item.dayOfInterview | date:'yyy-MM-dd'}} </td>
-          <td>{{item.comment}} </td>
-          <td>
-            <div class="col-md-12 align-center">
-                <button alt="edit" title="Edit" id="edit-interview"
-                type="button" class="btn" (click)="toEditInterview(item.id)" [disabled]="item.finishProcess">
-                  <i style="color: black;" class="fas fa-pen fa-2x"></i>
+        <thead class="text-center title-bg">
+          <tr>
+            <th scope="col">Interviews</th>
+            <th scope="col">Date of the Interview</th>
+            <th scope="col">Comments</th>
+            <th scope="col">Actions</th>
+          </tr>
+        </thead>
+        <tbody class="text-center">
+          <tr *ngFor="let item of list_interview">
+            <td>{{item.statusCandidate.name}}</td>
+            <td>{{item.dayOfInterview | date:'yyy-MM-dd'}} </td>
+            <td>{{item.comment}} </td>
+            <td>
+              <button alt="edit" title="Edit" id="edit-interview"
+              type="button" class="btn" (click)="toEditInterview(item.id)" [disabled]="item.finishProcess">
+              <i style="color: black;" class="fas fa-pen fa-2x"></i>
               </button>
-            </div>
-          </td>
-        </tr>
+            </td>
+          </tr>
+        </tbody>
       </table>
-      <ng-template #elseBlock>
-        <label class="form-control"> This candidate doesn't have interviews </label>
-      </ng-template>
-        </div>
     </div>
+    <div class="row">
+        <ng-template #elseBlock>
+          <label style="margin-left: 1rem; margin-top: 1rem;">This candidate doesn't have interviews </label>
+        </ng-template>
+    </div>
+  </div>
+  <div class="form-group row" *ngIf="list_interview?.length > 0"> 
     <br>
-    <div *ngIf="list_interview?.length > 0" class="row">
-      <div class="col-md-12 ">
-        <button class="btn interview-btn-pm" id="buttonfinish" (click)="onOpenDialog($event); isFinish = false;"
-          [disabled]="isFinish">Finish all interviews</button>
-        <ejs-dialog id='dialog' #ejDialog header='Are you sure ...? '
-          content='If you finish the process you wont be able to add or edit' [target]='targetElement' width='300px'
-          [buttons]='buttons' [visible]='hidden'>
-        </ejs-dialog>
-      </div>
+    <div class="col-sm-12">
+      <button class="btn interview-btn-pm" id="buttonfinish" (click)="onOpenDialog($event); isFinish = false;"
+        [disabled]="isFinish">Finish all interviews</button>
```

Comment:
- When i hover the "Finish all interviews" button with the mouse it looks like something in the page resize, im testing this in chrome
- To me, the pencil icon looks too big and adds an unwanted vertical space, but this is a subjective point of view if you think it's better like it is now it is fine too 👍

![image](https://user-images.githubusercontent.com/23131424/77771027-8d492e00-701c-11ea-96d5-a3651a2b0534.png)

Reply:
> * When i hover the "Finish all interviews" button with the mouse it looks like something in the page resize, im testing this in chrome
> * To me, the pencil icon looks too big and adds an unwanted vertical space, but this is a subjective point of view if you think it's better like it is now it is fine too +1
> 
> ![image](https://user-images.githubusercontent.com/23131424/77771027-8d492e00-701c-11ea-96d5-a3651a2b0534.png)

Thank you for your suggestions :ok_hand:
I made the corresponding improvements to the buttons, in addition to reducing the size of the pencil icon.